samples/nextjs/scripts/update-graphql-fragment-data.ts

```diff
@@ -65,6 +65,8 @@ fetch(jssConfig.graphQLEndpoint, {
         }
 
         console.log('GraphQL Fragment types successfully extracted!');
+
+        require('./fetch-graphql-introspection-data');
```

Comment:
This isn't really obvious added here. I was originally thinking have a single "update-graphql-data.ts" which does both fragment and introspection, which would avoid a bit over overlap with `generateConfig()`.
If you think these make more sense separate, the package.json "graphql:update" should call each separately.

Reply:
Created a separate command in package.json and call together